common/interpreter/wawaka_wasm/cdi/WawakaCDI.cpp

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2020 Intel Corporation
```

Comment:
so... as i understand it... you've opened up a bunch of ecalls that need to be run every time we initialize the interpreter (which is pretty much every time). why are these here? i thought we agreed that we were putting all policy in the eservice. and is there a particular reason why we aren't just passing these in on initialization (we already have a big JSON blob that we're passing in for the config). This really seems to complicate things considerably.

Reply:
As I've described previously (and above), WawakaCDI is a standalone tool that generates the compilation report. It's only used when the contracts are built, and other than being compiled when the interpreter is compiled, has nothing to do with the interpreter.